content_scripts/vimium_frontend.coffee

```diff
@@ -21,8 +20,8 @@ isEnabledForUrl = true
 passKeys = null
 keyQueue = null
 # The user's operating system.
-currentCompletionKeys = null
-validFirstKeys = null
+currentCompletionKeys = ""
+validFirstKeys = ""
```

Comment:
> Is there a good reason to do this?

This was changed only to fix the tests.  One of the DOM tests bubbles a key through the handler stack (which previously was empty, but now contains normal mode) without having done the necessary initialisation.

I'll go back and look at fixing the test properly.